### DISCUSSION TITLE
Viewer - Editor Mode

### DISCUSSION THREAD

**Discussion by ilkersigirci** [ssssota/svelte-exmarkdown#34] [Ideas]
Firstly, thank you for this library. It is simple to use and highly extendible. The current side by side view is good but I wonder if you plan to add Editor only or Viewer only mode in the future like in the [bytemd](https://github.com/bytedance/bytemd)

**Comment by ssssota** [comment]
Thanks for your comments!
Side by Side view like demo site is for demo only. 
This library provides only feature of markdown viewer.
If you want editor like demo site, you can build it yourself!

**Comment by ilkersigirci** [comment]
Oh, thank you for the quick response. I am currently trying to build [obsidian](https://obsidian.md/) like interface. I will look into your suggestion.
